libtsuba/src/RDGPartHeader.h

```diff
@@ -151,6 +151,7 @@ class KATANA_EXPORT RDGPartHeader {
       const std::string& file);
 
   bool IsEntityTypeIDsOutsideProperties() const;
+  bool IsUint16tEntityTypeIDs() const;
```

Comment:
nit: I'd prefer an enum because it's not clear what false implies. At least add doxygen comment to say that false implies uint8 type-ids.

It's fine to do this in a subsequent PR.